sequence/tests/test_sequence_manager.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+from unittest import mock
+
+import sequence.dynamo_sequence
+from sequence.sequence_manager import IdGenerator
+
+
+class TestSequenceManager(unittest.TestCase):
+
+    def setUp(self):
+        self.id_generator = IdGenerator()
+
+    @mock.patch.object(sequence.dynamo_sequence.DynamoSequenceGenerator, 'next')
+    async def test_generate_message_id(self, mock_next):
+        mock_next.return_value = 1
+        message_id = await self.id_generator.generate_transaction_id()
+        self.assertEquals(message_id, 1)
```

Comment:
Since we've got a unit test for this now it would be worth checking that the mock is called with the expected key:

mock.assert_called_with(expected_key)

https://realpython.com/python-mock-library/#assertions-and-inspection

Reply:
You also might need to use the `@test_utilities.async_test` annotation for these to work

Reply:
done